index.html

```diff
@@ -2271,8 +2274,14 @@ <h2 id="docconformance">
               [^th^]
             </td>
             <td>
-              <code>role=<a href="#index-aria-columnheader">columnheader</a></code> or
-              <a href="#index-aria-rowheader">`rowheader`</a>
+              <p>
+                If a descendant of a `table` element, <code>role=<a href="#index-aria-columnheader">columnheader</a></code> or
+                <a href="#index-aria-rowheader">`rowheader`</a> or <a href="#index-aria-rowheader">`cell`</a> (if not a header).
```

Comment:
we need to verify this, specifically if browsers will expose a `th` as a `cell` under different conditions, or if it's consistently exposed as a header.

if the former, then this is good.  if the latter, then we need to determine if that's appropriate or not.  and if not, then this becomes an issue for updating HTML AAM as well.